tools/test.py

```diff
@@ -138,6 +142,20 @@
             # Print available tests in order and exit
             print_tests(tests, options.format)
             sys.exit(0)
+        elif options.export:
+            base_source_paths = options.source_dir
+            # Default base source path is the current directory
+            if not base_source_paths:
+                base_source_paths = ['.']
```

Comment:
This base source path logic is done a few lines below this when compiling. Maybe we can merge these two bits? I'd prefer not to do that in two different places if we can avoid it.